queries.sql

```diff
@@ -68,3 +70,25 @@ WHERE date_of_birth
 BETWEEN '1990-01-01' 
 AND '2000-12-31' 
 GROUP BY species;
+
+-- Project 3
+--Join Query by owner name
+select * from animals JOIN owners on animals.owner_id=owners.id where owners.full_name='Melody Pond';
+
+--Join Query by species name
+select * from animals JOIN species on animals.species_id=species.id where species.name='Pokemon';
```

Comment:
- [ ] Please 🙏🏽 correct the print screen in your PR, it is supposed to show only the Pokemon list.

Reply:
corrected

![Screenshot (54)](https://github.com/Ademola-coding/vet_clinic/assets/96092850/7db81713-ccdc-41ac-9555-d0ccbad2dafc)